### PR TITLE
Adding new test case

### PR DESCRIPTION
**Pull request by mtipnis**

### CODE REVIEW

**Comment by mtipnis** [comment]
Will rename the file if there are no changes along with deletion of the older test.

**Comment by mtipnis** [comment]
Will rename the file if there are no changes along with deletion of the older test.

**Comment by mtipnis** [comment]
now ready for merge

File: modules/engine/test/mock/nested-limit-test.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "ebay": {
+        "apikey": "Qlio1a92e-fea5-485d-bcdb-1140ee96527"
+
+    },
+    "maxNestedRequests" : 2,
+    "maxRequestLength" : 10000000,
+    "maxResponseLength" : 10000000
+}
+
```
Comment (s3u) [inline modules/engine/test/mock/nested-limit-test.json:6]: This looked like config data, but it seems this is just some test data. Can you clarify?

File: modules/engine/test/nested-limit-test.js [hunk 2]
```diff
@@ -14,136 +14,65 @@
  * limitations under the License.
  */
 
-"use strict";
```
Comment (s3u) [inline modules/engine/test/nested-limit-test.js:17]: Please include include 'use strict'.

**Pull request merged**